src/component/Marketplace/index.jsx

```diff
@@ -7,6 +7,49 @@ import CollectionCard from '../Home/Cards/CollectionCard';
 import DiscoverCard from '../Home/Cards/DiscoverCard';
 
 function Marketplace() {
+  const inputRef = useRef();
+  const [nftData, setNftData] = useState(discoverData);
+  const [collectionData, setCollectionData] = useState(collection);
+
+  function filterNftData() {
+    const value = inputRef.current.value.toLowerCase();
+    // console.log(value);
+    const filterNft = nftData.filter((item) => {
+      return item.imgTitle.toLowerCase().includes(value);
+    });
+
+    const filterCollection = collectionData.filter((item) => {
+      return item.design.toLowerCase().includes(value);
+    });
+
+    if (value === '') {
+      setNftData(discoverData);
+      setCollectionData(collection);
+    } else {
+      setNftData(filterNft);
+      setCollectionData(filterCollection);
+    }
+  }
+
+  function getResult(e) {
+    const value = inputRef.current.value.toLowerCase();
+    // console.log(value);
+    const filterNft = nftData.filter((item) => {
```

Comment:
Why are you filtering from both nft and collection? Better to somehow check conditionally which tab is active and based on that filter